packages/react-percy-ci/src/index.js

```diff
@@ -16,10 +16,10 @@ export default async function run(percyConfig, webpackConfig, percyToken) {
 
   const environment = new Environment();
   const jsFiles = getJsFiles(assets);
-  await each(jsFiles, async jsFile => {
+  await each(async jsFile => {
```

Comment:
This was the main bug I had introduced. Switching from `bluebird` to `promise-each` required a small change.